web/modules/custom/farm_page/src/Services/FarmPageService.php

```diff
@@ -138,7 +139,15 @@ public function getProductsInfomation($products) {
       $product_info['product_name'] = $product_name[0]['value'];
       // Get Images.
       $product_images_field = $product->get('field_images')->getValue();
-      $product_images_url = $this->getImagesUrlFromArray($product_images_field);
+      // Check If Featured Product.
+      $product_isfeatured_field = $product->get('field_isfeature')->getValue()[0]['value'];
```

Comment:
Should be $product_is_featured. If this field is used to determine if a product is featured, it should be easier for user to understand, for example:
- Boolean field "Featured Product" with "field_featured_product" machine name.

In the future please avoid this kind of field and machine name.

Further more, please check if the field exists (!empty($product->get('field_isfeature'))) before getting its value to avoid the system returning an error.

Reply:
fixed to:  
// Check If Featured Product.
      $product_is_featured = NULL;
      if (!empty($product->get('field_isfeature')->getValue())) {
        $product_is_featured = $product->get('field_isfeature')->getValue()[0]['value'];
      }
      if ($product_is_featured == 1) {
        $style = 'featured_product';
        $product_images_url = $this->getImagesUrlWithStyle($product_images_field, $style);
      }
      else {
        $product_images_url = $this->getImagesUrlFromArray($product_images_field);
      }